flake.nix

```diff
@@ -136,6 +136,7 @@
               toolchain
               rust-analyzer
               cargo-outdated
+              cacert
```

Comment:
Hm? Why is this needed?

Reply:
Running `cargo outdated` gave me an error until I added this. :woman_shrugging: